Applications/Games/SimCity (2013)/Origin/script.js

```diff
@@ -0,0 +1,19 @@
+include(["engines", "wine", "quick_script", "origin_script"]);
+
+var installerImplementation = {
+    run: function () {
+        new OriginScript()
+            .name("SimCity (2013)")
+            .editor("Electronic Arts")
+            .applicationHomepage("http://www.origin.com/")
+            .author("ZemoScripter")
+            .category("Games")
+            .wineVersion("3.19")
```

Comment:
I forgot to change that.

Reply:
Let me remark this again: we should really try to use the latest stable/devel/staging as much as possible to avoid fragmenting the installed Wine versions too much.

Reply:
I get it but with the latest version the update/installation crashes.